src/libYARP_dev/src/modules/ServerInertial/ServerInertial.cpp

```diff
@@ -221,6 +221,53 @@ bool ServerInertial::initialize_ROS()
     return success;
 }
 
+bool yarp::dev::ServerInertial::openDeferredAttach(yarp::os::Property& prop)
+{
+//    yarp::os::Value &subdevice = prop.find("subdevice");
+    yDebug( "Really nothing to do here?? Waiting for 'attach' command");
```

Comment:
Leaving the  debug here may be ok (I don't have a clear idea) but probably it make sense to say in the debug string that is something related to ServerInertial, otherwise people will enable the debug option and they will see a lot of printouts for which is difficult to identify the source.